lib/Runtime/ByteCode/OpLayoutsAsmJs.h

```diff
@@ -283,165 +283,167 @@ namespace Js
 #define Uint32x4LayoutType typename SizePolicy::RegSlotType
 #define Uint16x8LayoutType typename SizePolicy::RegSlotType
 #define Uint8x16LayoutType typename SizePolicy::RegSlotType
+#define LAYOUT_FIELDS_HELPER(x, y) x ## y
+#define LAYOUT_FIELDS_DEF(x, y) LAYOUT_FIELDS_HELPER(x, y)
 #define LAYOUT_TYPE_WMS_REG2(layout, t0, t1) \
     template <typename SizePolicy> struct OpLayoutT_##layout\
     {\
-        t0##LayoutType LAYOUT_PREFIX_##t0()0;\
-        t1##LayoutType LAYOUT_PREFIX_##t1()1;\
+        t0##LayoutType LAYOUT_FIELDS_DEF(LAYOUT_PREFIX_##t0(), 0);\
```

Comment:
>LAYOUT_FIELDS_DEF(LAYOUT_PREFIX_##t0(), 0);\ [](start = 23, length = 44)

Is this to have `()` in a field name (weird, why do we want that)? From the change I assume directly `##0` doesn't work, right?